connector/src/it/scala/com/datastax/spark/connector/sql/CassandraDataFrameSelectUdtSpec.scala

```diff
@@ -0,0 +1,105 @@
+package com.datastax.spark.connector.sql
+
+import com.datastax.spark.connector.SparkCassandraITFlatSpecBase
+import com.datastax.spark.connector.cluster.DefaultCluster
+import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.functions.col
+import org.scalatest.Matchers
+import org.scalatest.concurrent.Eventually
+
+class CassandraDataFrameSelectUdtSpec extends SparkCassandraITFlatSpecBase with DefaultCluster with Eventually with Matchers {
+  override lazy val conn = CassandraConnector(defaultConf)
+
+  conn.withSessionDo { session =>
+    createKeyspace(session)
+
+    session.execute(
+      s"""CREATE TYPE ${ks}.embedded(
+        |        a TEXT,
+        |        b INT
+        |    )""".stripMargin
+    )
+
+    session.execute(
+      s"""CREATE TABLE ${ks}.crash_test(
```

Comment:
I there a way tuples could be selected as well?

Reply:
Added Support for tuples as well. They were also broken.